app/src/main/java/org/mozilla/tv/firefox/webrender/YouTubeBackHandler.kt

```diff
@@ -68,10 +67,10 @@ class YouTubeBackHandler(private val engineView: EngineView?, private val activi
         }
     }
 
+    fun goBackBeforeYouTube() = engineView!!.handleYoutubeBack(getIndexToGoBackTo())
```

Comment:
The param is nullable, but looking more closely it probably shouldn't be, since the whole class won't function without it.  I'm going to give some thought to how to fix that.